mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -80,13 +80,8 @@ def case_open(self):
 
         for case in self:
             values = {'active': True}
-
-            stages = self.env['mgmtsystem.action.stage']
-            values['stage_id'] = stages.search([
-                ['is_ending', '=', False],
-                ['is_starting', '=', False]
-            ]).id
-
+            values['stage_id'] = self.env.ref(
+                'mgmtsystem_action.stage_open').id
```

Comment:
It might be worth taking a look at what was made on the port to 9.0.
Maybe backporting those changes is a sensible thing to do.